lapack.go

```diff
@@ -11,22 +11,57 @@ const None = 'N'
 type Job byte
 
 const (
-	All       (Job) = 'A'
-	Slim      (Job) = 'S'
-	Overwrite (Job) = 'O'
+	All       Job = 'A'
+	Slim      Job = 'S'
+	Overwrite Job = 'O'
 )
 
 type CompSV byte
 
 const (
-	Compact  (CompSV) = 'P'
-	Explicit (CompSV) = 'I'
+	Compact  CompSV = 'P'
+	Explicit CompSV = 'I'
 )
 
+// Complex128 defines the complex128 interface for the Lapack function. This interface
```

Comment:
In this case it appears to be referring to a generic lapack function described by one of the interface methods. It could be reworded to be clearer - I think interface is being overloaded here (the method of the Go interface is describing the relevant LAPACK function interface).

Maybe:

```
Complex128 defines the public complex128 LAPACK API supported by gonum.
```